pom.xml

```diff
@@ -63,7 +63,7 @@
     </modules>
 
     <properties>
-        <geomesa.version>3.1.0</geomesa.version>
+        <geomesa.version>3.2.0-SNAPSHOT</geomesa.version>
```

Comment:
Are there any JTS or Spatial4J versions that we ought to double-check?

Reply:
haha i'm sure there are, given the recent issues we've seen... but I don't think we need to for this PR